swan/Chart.lock

```diff
@@ -1,9 +0,0 @@
-dependencies:
```

Comment:
Why are we deleting the Lock? Lock files are important to ensure consistency.

Reply:
Maybe you are used to `helm dependency build`.
I typically use `helm dependency update`, which creates the `Charts.lock` according to the dependencies expressed in `Charts.yaml`.

I am not aware of differences between the two, besides personal preferences.
It might be that the `.lock` goes out of sync with the `.yaml` in case dependencies (or their versions) are updated in the `.yaml`. Weird that they use this sort of double source of truth....

All in all, tell me what you prefer to use and I will make it work ;)

Reply:
I don't _really_ care which one we use, but I have the experience from other places (like npm, in the javascript world), in which the lock file is important to ensure that we pull always the same dependencies, thus guaranteeing that things work. 
Here our charts don't have indirect dependencies, so when you update, the lock only contains direct dependencies, so it's not that important. It was more a matter of principle, but I have no idea if publishing the chart relies somewhat on these lock files or not.

Reply:
Reverted with 0560d1366b818ba5472ba729a120531961813c01